src/histogram.cc

```diff
@@ -0,0 +1,111 @@
+#include "histogram.h"
+#include "histogram-inl.h"
```

Comment:
Should this be only `histogram-inl.h` or perhaps have an lint exclude (`// NOLINT(build/include_inline)`)?

Reply:
Yeah, thanks … feels a bit silly that you have to have a linter exception for cases like these, though?